app/models/katello/pool.rb

```diff
@@ -24,8 +24,8 @@ class Pool < Katello::Model
     scope :upstream, -> { where.not(upstream_pool_id: nil) }
     scope :redhat, -> { joins(:products).merge(Katello::Product.redhat).distinct }
 
-    include Glue::Candlepin::Pool
     include Glue::Candlepin::CandlepinObject
+    include Glue::Candlepin::Pool
```

Comment:
The order changed here so that the override of `import_candlepin_record` is picked up